net-proxy/trojan/trojan-1.11.0.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
You should use repoman to commit, so it fixes this old header for you automatically :)